### PR TITLE
Responsive navbar footer

### PR DESCRIPTION
**Pull request by Samah-Mustafa**
@basil-kawak @maher-suleyman Before reviwing I'm going to fix some issues like classes names

### CODE REVIEW

**Comment by Samah-Mustafa** [comment]
@louisrli PTAL

File: src/components/common/LangDropdown.jsx [hunk 1]
```diff
@@ -9,8 +9,8 @@ function LangDropdown() {
 
   return (
     <Dropdown className="langDropdown">
-      <Dropdown.Toggle id="dropdown-basic" className="langDropdown">
-        <img className="langPic" alt="lang. selector" src={lang} />
+      <Dropdown.Toggle id="dropdown-basic" className="Language-icon">
```
Comment (louisrli) [inline src/components/common/LangDropdown.jsx]: camel case class name

File: src/components/common/NavBar.jsx [hunk 2]
```diff
@@ -5,62 +5,94 @@ import { Link } from "react-router-dom";
 import LangDropdown from "./LangDropdown";
 
 export const NavBar = () => {
+  const [navItemsPosition, setNavItemsPosition] = React.useState("-100%");
```
Comment (louisrli) [inline src/components/common/NavBar.jsx]: not fixed from earlier comments

File: src/components/common/NavBar.jsx [hunk 3]
```diff
@@ -5,62 +5,94 @@ import { Link } from "react-router-dom";
...
+
+  const handleHumburger = () => {
```
Comment (louisrli) [inline src/components/common/NavBar.jsx]: hamburger typo

**Review by louisrli — CHANGES_REQUESTED** [review]

**Comment by Samah-Mustafa** [comment]
@louisrli PTAL

**Comment by basil-kawak** [comment]
@louisrli PTAL

File: src/components/common/NavBar.jsx [hunk 4]
```diff
@@ -5,62 +5,94 @@ import { Link } from "react-router-dom";
 import LangDropdown from "./LangDropdown";
 
 export const NavBar = () => {
+  const [isHamburgerShown, setIsHamburgerShown] = React.useState("-100%");
+
+  const handlehamburger = () => {
```
Comment (louisrli) [inline src/components/common/NavBar.jsx]: camelcase

File: src/components/common/NavBar.jsx [hunk 5]
```diff
@@ -5,62 +5,94 @@ import { Link } from "react-router-dom";
 import LangDropdown from "./LangDropdown";
 
 export const NavBar = () => {
+  const [isHamburgerShown, setIsHamburgerShown] = React.useState("-100%");
```
Comment (louisrli) [inline src/components/common/NavBar.jsx]: still same thing

Comment (louisrli) [reply src/components/common/NavBar.jsx]: this should be a boolean

**Review by louisrli — CHANGES_REQUESTED** [review]

File: src/containers/App.jsx [hunk 6]
```diff
@@ -16,13 +16,13 @@ function App() {
   return (
     <React.Fragment>
       <Router>
-        learn react
+        {/*learn react*/}
```
Comment (basil-kawak) [inline src/containers/App.jsx]: delete this

**Review by basil-kawak — APPROVED** [review]
good job

**Comment by Samah-Mustafa** [comment]
@louisrli PTAL

File: src/components/common/LangDropdown.jsx [hunk 7]
```diff
@@ -9,8 +9,8 @@ function LangDropdown() {
 
   return (
     <Dropdown className="langDropdown">
-      <Dropdown.Toggle id="dropdown-basic" className="langDropdown">
-        <img className="langPic" alt="lang. selector" src={lang} />
+      <Dropdown.Toggle id="dropdownBasic" className="LanguageIcon">
```
Comment (louisrli) [inline src/components/common/LangDropdown.jsx]: camelcase class name

**Review by louisrli — APPROVED** [review]
fix classname then its ok

**Pull request merged**